validation/telemetry/sample_v4_ping.json

```diff
@@ -10824,7 +10824,15 @@
             "key1": false,
             "key2": true
           }
-        }
+        },
+        "events": [
+          [2147, "ui", "click", "back_button"],
+	  [2213, "ui", "search", "search_bar", "google"],
```

Comment:
@fedepad you're mixing a tab with whitespaces on that line. Just use whitespaces and you should be fine! :-)